source/plugin/placeholders/creatingtokens.rst

```diff
@@ -78,10 +81,12 @@ Example: Player Location PlaceholderParser
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 This ``PlaceholderParser`` attempts to get the player's location in the world. If used without a ``Player`` as the 
-associated object, it returns an empty ``Component``. This implements the ``PlaceholderParser`` interface directly.
+associated object, it returns an empty :javadoc:`Component`. This implements the ``PlaceholderParser`` interface directly.
```

Comment:
As a note, only the first usage of a type on a page is Javadoc'd. Second (and subsequent) usages of `Component` should just be formatted as code. This affects other files too.

ref: https://github.com/SpongePowered/SpongeDocs/pull/834#discussion_r419994015

Reply:
Fixed